gef.py

```diff
@@ -2305,32 +2317,32 @@ class ARM(Architecture):
     syscall_register = "$r7"
     syscall_instructions = ["swi 0x0", "swi NR"]
 
-    def is_thumb(self):
+    def is_thumb(self) -> bool:
         """Determine if the machine is currently in THUMB mode."""
-        return is_alive() and gef.arch.register(self.flag_register) & (1 << 5)
+        return is_alive() and gef.arch.register(self.flag_register) & (1 << 5)  # TODO: get_register can return None
```

Comment:
You can add this TODO to the project instead of in the code.

Reply:
oh, i thought i removed all of my todo comments before pushing. sorry, this PR is a little messed up. Too many lines to look at...

Reply:
added the todos to the linked project 11